src/tutorial/testing.md

```diff
@@ -46,6 +46,10 @@ Rust has a built-in test framework,
 so let's start by writing a first test:
 
 ```rust,ignore
+# fn answer() -> i32 {
+#   42
+# }
+#
```

Comment:
> Add default hidden code

Does mdbook hide `#` prefixed lines like rusutdoc?

Reply:
Yes

Reply:
https://rust-lang.github.io/mdBook/format/theme/syntax-highlighting.html#hiding-code-lines for reference